easybuild/easyconfigs/l/LAST/LAST-869-intel-2017a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'MakeCp'
+
+name = 'LAST'
+version = '869'
+
+homepage = 'http://last.cbrc.jp/'
+description = """ LAST finds similar regions between sequences. The main technical innovation
+ is that LAST finds initial matches based on their multiplicity,
+ instead of using a fixed length (e.g. BLAST uses 11-mers)."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'cstd': 'c++11', 'opt': True}
+
+source_urls = ['http://last.cbrc.jp/']
+sources = ['%(namelower)s-%(version)s.zip']
+
+patches = ['%(name)s-%(version)s_makefile.patch']
+
```

Comment:
please drop this empty line, and the one above as well

Reply:
@boegel: done